regnety/dataset/imagenet.py

```diff
@@ -162,10 +192,25 @@ def color_jitter(self, image: tf.Tensor, target: tf.Tensor) -> tuple:
         aug_images = tf.image.random_hue(aug_images, hue_delta)
         aug_images = tf.image.random_saturation(aug_images, saturation_lower, 
             saturation_upper)
-        aug_images = tf.image.random_flip_left_right(aug_images)
+        
         
         return aug_images, target
 
+    def random_flip(self, image: tf.Tensor, target: tf.Tensor) -> tuple:
+        """
+        Returns randomly flipped batch of images. Only hotizontal flip 
+        is available
+
+        Args: 
+            image: Batch of images to perform random rotation on.
+            target: Target tensor.
+
+        Returns:
+            Augmented example with batch of images and targets with same dimensions.
+        """
+
+        aug_images = tf.image.random_flip_left_right(image)
```

Comment:
Can't we map this directly? Why a separate function for it?

Reply:
We can, but that'll require a lambda function. No technical reason, I just wanted avoid clutter in `make_dataset()`.